settings/Core.setting.php

```diff
@@ -732,12 +732,8 @@
     'group' => 'core',
     'name' => 'enable_components',
     'type' => 'Array',
-    'quick_form_type' => 'Select',
-    'html_type' => 'Select',
-    'html_attributes' => [
-      'multiple' => 1,
-      'class' => 'crm-select2',
-    ],
+    'quick_form_type' => 'CheckBoxes',
+    'html_type' => 'crm-checkbox-list',
```

Comment:
@colemanw better add this to the setting spec if we have a new type

Reply:
@eileenmcnaughton which spec is that?

Reply:
https://docs.civicrm.org/dev/en/latest/framework/setting/

Reply:
Hmm, those docs suggest that `html_type` is preferred over `quickform_type` but when I was stepping through the code, `html_type` was getting ignored by the buildForm fn, which is why I felt free to make up something new and stick it in there.

Buuuuut,

Maybe instead of making up something new, we should just apply this styling to all checkboxes on settings pages. It's nicer that what's there already. Grep shows seven settings other than this one of type checkboxes.

Reply:
Yeah - that makes sense. Probably we do prefer html_type since it makes more sense but the code couldn't get there all in one go...